images/hub/jupyterhub_config.py

```diff
@@ -254,18 +264,19 @@ def generate_user_name(spawner):
 c.KubeSpawner.environment.update(get_config('singleuser.extra-env', {}))
 
 # Enable admins to access user servers
-c.JupyterHub.admin_access = get_config('auth.admin.access')
+c.JupyterHub.admin_access = get_config('auth.admin.access', False)
 c.Authenticator.admin_users = get_config('auth.admin.users', [])
 c.Authenticator.whitelist = get_config('auth.whitelist.users', [])
 
+set_config_if_not_none(c.JupyterHub, 'base_url', 'hub.base_url')
 c.JupyterHub.base_url = get_config('hub.base_url')
 
 c.JupyterHub.services = []
 
 if get_config('cull.enabled', False):
-    cull_timeout = get_config('cull.timeout')
-    cull_every = get_config('cull.every')
-    cull_concurrency = get_config('cull.concurrency')
+    cull_timeout = get_config('cull.timeout', 3600)
+    cull_every = get_config('cull.every', 600)
+    cull_concurrency = get_config('cull.concurrency', 10)
```

Comment:
Should we perhaps remove the default entry in the values file while we add the default here? If not, this default will be ignored forever assuming we are running the helm chart.

Reply:
related: #341

Reply:
Perhaps we should do this systematically when fixing #341 though, it would reduce risks of me messing up with rebasing a lot of work already done.

Reply:
Yes, that should be a separate PR. This one is just fixing a bug when things were undefined in the configmap.